imagefiles/entrypoint.sh

```diff
@@ -35,6 +35,11 @@ if [[ -n $BUILDER_UID ]] && [[ -n $BUILDER_GID ]]; then
         /dockcross/pre_exec.sh
     fi
 
+    # Execute project specific pre execution hook
+    if [[ -e /work/.dockcross ]]; then
+       /work/.dockcross
```

Comment:
@thewtex  I am wondering if this should this be `exec chpst -u :$BUILDER_UID:$BUILDER_GID /work/.dockcross` ?